components/ImageSlider.tsx

```
@@ -0,0 +1,76 @@
+            className="btn btn-dark position-absolute top-50 start-0 translate-middle-y m-2"
+            onClick={goToPrevious}
+            style={{ zIndex: 1 }}
+          >
+            <IoChevronBack size={24} color="white" />
+          </button>
+
+          <button
+            className="btn btn-dark position-absolute top-50 end-0 translate-middle-y m-2"
+            onClick={goToNext}
+            style={{ zIndex: 1 }}
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)